### PR TITLE
[doc] Add coding style to CONTRIBUTING.

### PR DESCRIPTION
**Pull request by ejgallego**
As suggested by @rgrinberg.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -70,3 +70,9 @@ your commit automatically with `git commit -s`.
 
 [dco]: http://developercertificate.org/
 [ocl]: http://ocamllabs.io/
+
+Coding style
+------------
+
+- we wrap at 80 characters,
+- we prefer `[Ss]nake_case` over `[Pp]ascalCase]`.
```
Comment (diml) [inline CONTRIBUTING.md]: there is a trailing `]`:

```suggestion
- we prefer `[Ss]nake_case` over `[Pp]ascalCase`.
```

Comment (ejgallego) [reply CONTRIBUTING.md]: I am not sure what the meaning of the brackets is, @rgrinberg wrote it like that.

Comment (rgrinberg) [reply CONTRIBUTING.md]: The suggestion is valid. It was just a typo. I'd also remove the `we` in both points here. As it's already clear that these are all recommendations from the team.

**Comment by emillon** [comment]
Thanks!

**Pull request merged**